keyboards/karn/info.json

```diff
@@ -0,0 +1,69 @@
+{
+    "manufacturer": "Robert Mills",
+    "keyboard_name": "karn",
+    "maintainer": "robcmills",
+    "features": {
+        "bootmagic": true,
+        "command": false,
+        "console": false,
+        "extrakey": true,
+        "mousekey": true,
+        "nkro": true
+    },
+    "development_board": "blackpill_f401",
+    "url": "https://github.com/robcmills/karn-keyboard",
+    "usb": {
+        "device_version": "1.0.0",
+        "pid": "0x0000",
+        "vid": "0xFEED"
+    },
```

Comment:
```suggestion
    },
    "bootmagic": {
        "matrix": [0, 1]
    },
    "split": {
        "enabled": true,
        "bootmagic": {
            "matrix": [4, 4]
        }
    },
```

Reply:
So it looks like there is a general move from the `config.h` to conveying everything in `info.json`. Is that an accurate description of the current state of qmk?